Examples/Algorithms/Propagation/include/ActsExamples/Propagation/PropagationAlgorithm.hpp

```diff
@@ -104,6 +104,8 @@ class PropagationAlgorithm : public BareAlgorithm {
     std::pair<double, double> ptRange = {100_MeV, 100_GeV};
     /// looper protection
     double ptLoopers = 500_MeV;
+    /// Search tolerance
+    double searchTolerance = 0_mm;
```

Comment:
The same `searchTolerance` in local x and y direction? Shall we have two values for x and y separately?